tracer/build/_build/Build.cs

```diff
@@ -412,11 +412,11 @@ void DeleteReparsePoints(string path)
                     .When(!string.IsNullOrEmpty(NugetPackageDirectory), o => o.SetPackageDirectory(NugetPackageDirectory))
                 );
 
-            var (framework, runtimes) = IsOsx switch
-            {
-                true => (TargetFramework.NETCOREAPP3_1, "net6.0"),
-                false => (TargetFramework.NET6_0, "net472 netcoreapp3.1.0"),
-            };
+                var (framework, runtimes) = IsOsx switch
+                {
+                    true => (TargetFramework.NETCOREAPP3_1, "net6.0"),
```

Comment:
wait, is this right 🤔 
```suggestion
                    true => (TargetFramework.NETCOREAPP3_1, "netcoreapp3.1"),
```

Reply:
yes because there's no .NET Core 3.1 for OSX (at least for ARM64) version

Reply:
I'll no click in you suggestions. :trollface: